sopel/plugins/handlers.py

```diff
@@ -75,33 +77,33 @@ class AbstractPluginHandler(object):
     on shutdown (either upon exiting Sopel or unloading that plugin).
     """
     def load(self):
-        """Load the plugin
+        """Load the plugin.
 
         This method must be called first, in order to setup, register, shutdown,
         or configure the plugin later.
         """
         raise NotImplementedError
 
     def reload(self):
-        """Reload the plugin
+        """Reload the plugin.
 
         This method can be called once the plugin is already loaded. It will
         take care of reloading the plugin from its source.
         """
         raise NotImplementedError
 
     def get_label(self):
-        """Retrieve a display label for the plugin
+        """Retrieve a display label for the plugin.
 
-        :return: A human readable label for display purpose
+        :return: a human readable label for display purpose
         :rtype: str
 
-        This method should, at least, return ``module_name + S + "plugin"``.
+        This method should, at least, return ``<module_name> plugin``.
```

Comment:
Isn't this what I wanted when first reviewing `plugins` in #1479? 😂 The PR review only seems to have a question (https://github.com/sopel-irc/sopel/pull/1479#discussion_r277106408) but I recall a fairly in-depth discussion about it on IRC.

Reply:
ikr

In the long term, practicality beats purity. I don't like it, but that's how we roll and it's better.